src/components/Footer/Footer.js

```diff
@@ -1,26 +1,53 @@
-import './Footer.css'
+import styled from 'styled-components'
 
 function Footer () {
     return (
         <footer>
-            <nav className="footer__nav">
-                <button className="footer__button">
+            <FooterContainer>
+                <FooterButton>
                 Home
         
-                </button>
+                </FooterButton>
 
-                <button className="footer__button">
+                <FooterButton>
                 Create
 
-                </button>
+                </FooterButton>
 
-                <button className="footer__button">
+                <FooterButton>
                 Profile
 
-                </button>
-            </nav>
+                </FooterButton>
+            </FooterContainer>
         </footer>
     )
 }
 
-export default Footer
\ No newline at end of file
+const FooterContainer = styled.nav`
+position: fixed;
+bottom: 0px;
+width: 100%;
+height: 8%;
+display: flex;
+z-index: 2;
+box-shadow: 0px 0px 5px 5px rgba(0, 0, 50, 0.75);
+-webkit-box-shadow: 0px 0px 5px 5px rgba(0, 0, 50, 0.75);
+-moz-box-shadow: 0px 0px 5px 5px rgba(0, 0, 0, 50.75);
```

Comment:
There's a bunch of properties where you don't need to put `-webkit-` and `-moz-` anymore, as most properties are widely implemented in modern browsers. As far as I can tell, you only need these still when you want to make sure, very old browsers can render certain properties.

Reply:
Also on a side note: styled-components adds most of these for us anyways 🙂